Rakefile

```diff
@@ -20,3 +20,78 @@ end
 
 desc 'Run bundle-audit'
 Bundler::Audit::Task.new
+
+namespace :doc do
```

Comment:
generates a `docs` folder with all known wsdl's.
i found this super useful being able to search these things when working through this process and figured it might be nice for others as well.